Mathlib/GroupTheory/Torsion.lean

```diff
@@ -138,15 +138,15 @@ theorem ExponentExists.isTorsion (h : ExponentExists G) : IsTorsion G := fun g =
       "The group exponent exists for any bounded additive torsion group."]
 theorem IsTorsion.exponentExists (tG : IsTorsion G)
     (bounded : (Set.range fun g : G => orderOf g).Finite) : ExponentExists G :=
-  exponentExists_iff_ne_zero.mpr <|
+  exponent_ne_zero.mp <|
     (exponent_ne_zero_iff_range_orderOf_finite fun g => (tG g).orderOf_pos).mpr bounded
 #align is_torsion.exponent_exists IsTorsion.exponentExists
 #align is_add_torsion.exponent_exists IsAddTorsion.exponentExists
 
 /-- Finite groups are torsion groups. -/
 @[to_additive is_add_torsion_of_finite "Finite additive groups are additive torsion groups."]
 theorem isTorsion_of_finite [Finite G] : IsTorsion G :=
-  ExponentExists.isTorsion <| exponentExists_iff_ne_zero.mpr exponent_ne_zero_of_finite
+  ExponentExists.isTorsion <| exponent_ne_zero.mp exponent_ne_zero_of_finite
```

Comment:
```suggestion
  ExponentExists.of_finite.isTorsion
```

Reply:
Why not use dot notation all the way?

Reply:
`_root_.ExponentExists.isTorsion` doesn't work in dot notation because it's about `Monoid.ExponentExists`. (I don't want to deal with that in this PR.)

Reply:
Oh yeah that's why